js/ui/projectManager.js

```diff
@@ -505,12 +515,66 @@ define([
       if (!keepRetained) chrome.storage.local.remove("retainedProject");
       this.render();
     },
-    
+
     getPaths: function() {
       return Object.keys(this.pathMap);
     }
   };
 
+  // Resize sidebar
+  // min / max sidebar width
+  var MIN_WIDTH = 100;
+  var MAX_WIDTH = 500;
+
+  function setupSidebarResize(element, document) {
+    // Sidebar resizing
+    var isResizing = false;
+    // make sure width is not set in element `style`
+    element.style.width = null;
+
+    var startSidebarResize = function (evt) {
+      if (evt.target.hasClass('project-resizer')) {
+        // do not resize when 'autohide' is on
+        if (element.hasClass('autohide')) return;
+
+        evt.preventDefault();
+        evt.stopPropagation();
+
+        element.addClass('_resize');
+        isResizing = true;
+      }
+    };
+
+    var doSidebarResize = function (evt) {
+      if (!isResizing) return;
+
+      var mouseX = evt.pageX;
```

Comment:
Yep, `pageX` is relative to the document, while `clientX` is relative to `viewport` (same thing in caret currently, but if it ever starts to matter - `pageX` will be more correct usage)